src/components/Discussion/Composer/CommentComposer.js

```diff
@@ -186,6 +191,7 @@ export const CommentComposer = props => {
   const onChangeText = ev => {
     const nextText = ev.target.value
     setText(nextText)
+    setLabel(getLabel(nextText))
```

Comment:
`getLabel` might be not defined, right, because context comes "from outside"? Is it reasonable to harden this?

Reply:
```suggestion
        if (getLabel) {
      setLabel(getLabel(nextText))
    }
```

Reply:
```suggestion
    getLabel && setLabel(getLabel(nextText))
```